library/tiqr/Tiqr/Service.php

```diff
@@ -243,6 +247,9 @@ public function __construct(LoggerInterface $logger, array $options=array(), int
             throw new RuntimeException('No state storage configuration is configured, please provide one');
         }
         $this->_stateStorage = Tiqr_StateStorage::getStorage($options["statestorage"]["type"], $options["statestorage"], $logger);
+        // Set a default salt, with the SESSION_KEY_LENGTH_BYTES (16) length keys we're using a publicly
+        // known salt already gives excellent protection.
+        $this->_stateStorageSalt = $options["statestorage"]['salt'] ?? '8xwk2pFd';
```

Comment:
I'd rather see a hard coded value in a config file then in an implementation. But I can live with this.